pkg/utils/utils.go

```diff
@@ -24,11 +24,16 @@ func ShortHostname() (shortName string, err error) {
 	if err != nil {
 		panic(err)
 	}
-	splitHostname := strings.SplitN(hostname, ".", 2)
-	shortName = splitHostname[0]
+	shortName, err = GetShortHostname(hostname)
 	return shortName, err
 }
 
+func GetShortHostname(hostName string) (shortName string, err error) {
```

Comment:
Does return error is needed here? function always returns error=nil

Reply:
I removed that.